framework/src/main/java/org/checkerframework/framework/type/TypeFromTypeTreeVisitor.java

```diff
@@ -53,6 +54,15 @@ public AnnotatedTypeMirror visitAnnotatedType(AnnotatedTypeTree node, AnnotatedT
         List<? extends AnnotationMirror> annos = TreeUtils.annotationsFromTree(node);
 
         if (type.getKind() == TypeKind.WILDCARD) {
+            // In java 9, the type attached to the AnnotatedTypeTree node is a WildcardType
```

Comment:
clean up comments
file issue in eisop to repro issue in openjdk
so that we have a comment pointing to openjdk bug report